src/module.cpp

```diff
@@ -925,6 +925,24 @@ void Module::AddFunctionDeclaration(const std::string &name, const FunctionType
         function->addFnAttr(llvm::Attribute::NoInline);
     }
 
+    if (g->target_os == TargetOS::windows) {
+        // Enable generation an unwind table during codegen.
+        // It is needed to generate backtraces during debugging and to unwind callstack.
+#if ISPC_LLVM_VERSION <= ISPC_LLVM_13_0
+        llvm::AttrBuilder B;
+        B.addAttribute(llvm::Attribute::UWTable);
+        function->addAttributes(llvm::AttributeList::FunctionIndex, B);
+#elif ISPC_LLVM_VERSION <= ISPC_LLVM_14_0
+        llvm::AttrBuilder B(function->getContext());
```

Comment:
Is `AttrBuilder` really needed? Does simple `function->addFnAttr(llvm::Attribute::UWTable);` work?